fe/fe-core/src/main/java/com/starrocks/sql/StatementPlanner.java

```diff
@@ -181,6 +182,7 @@ public static boolean supportedByNewPlanner(StatementBase statement) {
                 || statement instanceof AnalyzeStmt
                 || statement instanceof CreateAnalyzeJobStmt
                 || statement instanceof CreateCatalogStmt
+                || statement instanceof CreateTableStmt
```

Comment:
Is it necessary to change CreateTableStmt to CreateTableStatement? @HangyuanLiu

Reply:
old code no need change name